src/data/sources.json

```diff
@@ -283,6 +283,80 @@
                     "title": "WorldBank"
                 },
                 "qualifier": "M: Male\nF: Female"
+            },
+            "allPopulationSexYears": {
+                "source": {
+                    "href": "https://www.knbs.or.ke/download/statistics-abstract-2018/",
+                    "title": "Statistical Abstract, 2018"
+                }
+            },
+            "allSecondarySchoolTotEnrollments": {
+                "source": {
+                    "href": "https://www.knbs.or.ke/download/statistics-abstract-2018/",
+                    "title": "Statistical Abstract, 2018"
+                }
+            },
+            "allPrimarySchoolTotEnrollments": {
+                "source": {
+                    "href": "https://www.knbs.or.ke/download/statistics-abstract-2018/",
+                    "title": "Statistical Abstract, 2018"
+                }
+            },
+            "allBudgetAllocations": {
+                "source": {
+                    "href": "https://www.knbs.or.ke/download/statistics-abstract-2018/",
+                    "title": "Statistical Abstract, 2018"
+                }
+            },
+            "allHealthFacilityBeds": {
+                "source": {
+                    "href": "https://www.knbs.or.ke/download/statistics-abstract-2018/",
+                    "title": "Statistical Abstract, 2018"
+                }
+            }
+        },
+        "level1": {
+            "allPopulationSex2009S": {
+                "source": {
+                  "href": "https://dc.sourceafrica.net/documents/21197-Census-2009.html",
+                  "title": "Census, 2009"
+                }
+             },
+            "allPopulationSexYears": {
+                "source": {
+                    "href": "https://www.knbs.or.ke/download/statistics-abstract-2018/",
+                    "title": "Statistical Abstract, 2018"
+                }
+            },
+            "allHivPrevalences": {
+                "source": {
+                    "href": "https://nacc.or.ke/wp-content/uploads/2018/12/HIV-estimates-report-Kenya-20182.pdf",
```

Comment:
Linking directly to PDF _ain't_ cool.

Reply:
@kilemensi  I will ping soila to upload this to openAfrica

Reply:
updated this